client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx

```diff
@@ -26,12 +26,10 @@ const NewInteractionEventDialog : React.FC<Props> = (props: Props) : JSX.Element
 
     const epidemiologyNumber = useSelector<StoreStateType, number>(state => state.investigation.epidemiologyNumber);
 
-    const canConfirm = React.useMemo<boolean>(() => true, [])
```

Comment:
why deleting it?

Reply:
I did not tamper with this line, not sure how it got deleted. This should be restored.

Reply:
Done.